invenio_accounts/templates/invenio_accounts/login_user.html

```diff
@@ -34,6 +34,7 @@
       <h3 class="text-center panel-free-title">{{_('Log in to account') }}</h3>
       {%- endblock form_header %}
       {%- block form_outer %}
+      {% block block_before_login_form %}{% endblock block_before_login_form %}
```

Comment:
you don't really need it. you can override `form_outer`:

```jinja2
{% block form_outer %}
  do something before login form
  {{ super() }}
{% endblock %}
```

--
http://jinja.pocoo.org/docs/dev/templates/#super-blocks